crates/fluvio-test/src/tests/mod.rs

```diff
@@ -72,6 +72,11 @@ impl TestRecordBuilder {
         self
     }
 
+    pub fn with_empty_data(mut self, data_size: usize) -> Self {
```

Comment:
It seems like that method is not used in the final version, so there is no sense in keeping it